arch/arm/boot/dts/stonehenge-big-common.dtsi

```diff
@@ -4,6 +4,15 @@
  *
  */
 
+&i2c2 {
```

Comment:
why is this codec being included on stonehenge big? (trail & aventura)

Reply:
Because this chip will also be mounted to Aventura and Trail as well in the  future

Reply:
> Because this chip will also be mounted to Aventura and Trail as well in the future

I've checked this and this will not happen as there are many changes to be done in order to replace component